app/scripts/common-transforms.js

```diff
@@ -174,6 +192,123 @@ var commonTransforms = (function(_) {
             });
 
             return arrayToReturn;
+        },
+
+        getClickableObjectArr: function (records, type) {
+            var result = [];
+            if(records) {
+                if(records.constructor === Array) {
+                    records.forEach(function(record) {
+                        if(record.name) {
+                            var obj = {
+                                _id: record.uri,
+                                _type: type,
+                                title: record.name,
+                                subtitle: ''
+                            };
+                            result.push(obj);
+                        }
+                    });
+                } else {
+                    var obj = {
+                        _id: records.uri,
+                        _type: type,
+                        title: records.name,
+                        subtitle: ''
+                    };
+                    result.push(obj);
+                }
+            }
+            
+            return result;
+        },
+
+        combineArrays: function(arr1, arr2) {
+            arr2.forEach(function(elem) {
+                arr1.push(elem);
+            });
+            return arr1;
+        },
+        offerTimelineData: function(data) {
+            var newData = {};
+
+            if(data.hits.hits.length > 0) {
+                var aggs = data.aggregations;
+                newData.offerTimeline = this.transformBuckets(aggs.offersPhone.offerTimeline.buckets, 'date', 'key_as_string');
+            }
+        
+            return newData;
+        },
+        offerLocationData: function(data) {
+            var newData = {};
+
+            if(data.hits.hits.length > 0) {
+                var aggs = data.aggregations;
+                newData.offerLocation = getGeoFromKeys(aggs.phone.city.buckets);
+            }
+        
+            return newData;
+        },
+         getSellerId: function(record) {
+            sellerId = '';
+            if(record.owner) {
+
+                if(_.isArray(record.owner)) {
+                    //phone will one seller 
+                    sellerId = record.owner[0].uri;    
+                }
+                else {
+                    sellerId = record.owner
+                }
+                
+            }
+
+            return sellerId;
+        },
+        getEmailAndPhoneFromMentions(mentions) {
```

Comment:
This is an error.  It should be `getEmailAndPhoneFromMentions: function(mentions) {`